support-frontend/assets/components/checkoutBenefits/checkoutBenefitsListContainer.tsx

```diff
@@ -123,9 +116,6 @@ export function CheckoutBenefitsListContainer({
 		),
 		handleButtonClick,
 		countryGroupId,
-		rowSpacingNarrow: useOptimisedMobileLayout,
-		cssOverrides: useOptimisedMobileLayout
-			? optimisedLayoutOverrides
-			: undefined,
+		cssOverrides: layoutOverrides,
```

Comment:
So we need to pass these style rules as `cssOverrides`? I can see the `cssOverrides` were added when set-up the first Mobile Optimisation test [here](https://github.com/guardian/support-frontend/commit/953c9e89237383d1733d4373d451c6cc3a1ad7a7#diff-9f7effdc73d8054d2d6b5b25467864c2e6c12e4de2c8a26244ba948bad8e2d23R126). Can these style rules just be moved to the default rules of the component?

Reply:
Agree these overrides simply are not needed. Have removed css overrides from containers->  benefitslist, pricecards and paymentmethods (all ones directly added within the original PR). The styles have been placed into their components. Completed, thanks.